ffmpeg/ffmpeg_test.go

```diff
@@ -1869,3 +1870,25 @@ func TestResolution_Clamp(t *testing.T) {
 	checkError = require.Error
 	test(l, Size{300, 600}, portrait, Size{300, 600})
 }
+
+func TestGetFPSDuration(t *testing.T) {
```

Comment:
this does not seem to actually test anything? please see how other testcases are constructed

Reply:
Rewrote into two tests which validate FPS and duration. Added test files. https://github.com/livepeer/lpms/pull/407/files#diff-aeb5ed5cb59cc9ff3ded5fa0bebee480bb3273b66c016f159c80c188a6b21611